frontend/src/index.tsx

```diff
@@ -13,6 +14,13 @@ import { loadPostHogJS } from './loadPostHogJS'
 loadPostHogJS()
 initKea()
 
+// On Chrome + Windows, the country flag emojis don't render correctly. This is a polyfill for that.
+// It won't be applied on other platforms.
+//
+// NOTE: The first argument is the name of the polyfill to use. This is used to set the font family in our CSS.
+// Make sure to update the font family in the CSS if you change this.
+polyfillCountryFlagEmojis('Emoji Flags Polyfill')
```

Comment:
I plan on using our own CDN to serve the font assets, but I need to figure out how to do it
https://posthog.slack.com/archives/C0113360FFV/p1741734888246059

Reply:
Nobody helped me, so I'll keep this as is now, we can fix it in the future